ddtrace/span.py

```diff
@@ -61,6 +75,7 @@ class Span(object):
         "_meta",
         "error",
         "_metrics",
+        "_store",
```

Comment:
```suggestion
        "_request_store",
```
store is a bit generic 🤷